libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/verification/RustSessionVerificationService.kt

```diff
@@ -188,6 +223,14 @@ class RustSessionVerificationService(
         _verificationFlowState.value = VerificationFlowState.Initial
     }
 
+    override suspend fun skipVerification(): Boolean = runCatching {
+        val existingSession = sessionStore.getSession(client.userId()) ?: error("No session with id ${client.userId()}")
+        sessionStore.updateData(existingSession.copy(needsVerification = false))
+        true
+    }
+        .onFailure { Timber.e(it, "Failed to skip verification") }
+        .getOrDefault(false)
+
```

Comment:
The returned boolean does not seem to be used, maybe this API should return `Result<Unit>` instead of `Boolean`?